src/components/Common/SkeletonLoader/SkeletonLoader.styl

```diff
@@ -0,0 +1,85 @@
+.skeletonLoader
+  box-sizing border-box
+  display flex
+  flex-direction column
+  justify-content space-between
+  gap 4px
+  position relative
+  width 100%
+  padding 4px
+  flex 0 0 auto
+  margin-left 0
+  z-index 1
+  color transparent
+  --skeleton-light-color linear-gradient(0deg, rgba(9, 109, 217, 0.05), rgba(9, 109, 217, 0.05)), #FAFAFA
+  --skeleton-dark-color linear-gradient(0deg, rgba(9, 109, 217, 0.14), rgba(9, 109, 217, 0.14)), #FAFAFA
```

Comment:
Can we put definitions of css vars to the top of the scope in which owns it?

Reply:
done - added a few more aswell